examples/ProtoBuffer/protobuffer.ino

```diff
@@ -51,31 +51,28 @@ void loop()
   byte *buffer;
   size_t size;
 
+  debugSerial.print("Motion : ");
+  debugSerial.print(motion);
+  debugSerial.print("; Humidity : ");
+  debugSerial.print(humidity);
+  debugSerial.print("; Water : ");
+  debugSerial.print(water);
+  debugSerial.print("; Temperature celcius : ");
+  debugSerial.print(temperature_celcius);
+  debugSerial.print("; Temperature farhenheit : ");
+  debugSerial.println(temperature_fahrenheit);
```

Comment:
Put some newlines instead of concatenating in one big string.

Also, I'm not sure if we want to print all this in the first place